src/plugin.tsx

```diff
@@ -37,13 +36,12 @@ export enum LiveBroadcastStates {
     Offline = "Offline"
 }
 
-export class KalturaLivePlugin implements OnMediaUnload, OnRegisterUI, OnMediaLoad, OnPluginSetup {
+export class KalturaLivePlugin implements OnMediaUnload, OnMediaLoad, OnPluginSetup {
     private _kalturaClient = new KalturaClient();
-    // represents if the entry type is a live entry
     private _isLiveEntry = false;
-    // represents if the entry is in live mode or unknown
     private _broadcastState: LiveBroadcastStates = LiveBroadcastStates.Unknown;
-
+    private _wasPlayed: boolean = false;
+    private _httpError: boolean = false;
```

Comment:
probably we should have generic error flag that triggered displaying of offline slate.